app/src/main/java/org/thoughtcrime/securesms/preferences/CorrectedPreferenceFragment.java

```diff
@@ -125,29 +125,6 @@ private boolean isBottom(int position) {
         return next instanceof PreferenceCategory;
       }
 
-      public Drawable getBackground(Context context, int position) {
```

Comment:
I'm guessing this doesn't break things for the other settings screens (chats, notifications etc all the other ones?)

Reply:
Your concern is well founded as this breaks all of the preference backgrounds.

Reply:
reverted... well mostly, I added ripple to those buttons 🎉